source/common/http/utility.cc

```diff
@@ -1033,5 +1033,42 @@ Utility::AuthorityAttributes Utility::parseAuthority(absl::string_view host) {
   return {is_ip_address, host_to_resolve, port};
 }
 
+envoy::config::route::v3::RetryPolicy
+Utility::convertCoreToRouteRetryPolicy(const envoy::config::core::v3::RetryPolicy& retry_policy,
+                                       const std::string& retry_on) {
+  envoy::config::route::v3::RetryPolicy route_retry_policy;
+  uint64_t base_interval_ms = 1000;
```

Comment:
How bout using std::chrono::microseconds(1000) so that the units are part of the type. I think this will also avoid a subsequent conversion down on line 1061/1063.

auto base_interval = std::chrono::microseconds(1000)
auto max_interval = base_interval * 10;

Reply:
I think that we don't have utility API to convert `chrono::milliseconds` to `protobuf::Duration`. But we have `MillisecondsToDuration` that converts `uint64_t` to `protobuf::Duration`. I think this is the easiest way.

Reply:
Oh, that's annoying! :) But fair enough.